packages/nitro-protocol/contracts/ninja-nitro/AdjudicatorFactory.sol

```diff
@@ -76,7 +76,7 @@ contract AdjudicatorFactory {
 
     /// @dev Allows us to create new channel contact using CREATE2
     function _deployChannelProxy(bytes32 channelId) internal returns (address) {
-        bytes32 salt = channelId;
```

Comment:
I am curious whether `solc` optimizes this away.

Reply:
I would have thought any compiler worth its salt would do that :-P (but I haven't checked)